api/src/controllers/login.js

```diff
@@ -92,9 +90,7 @@ const getEnabledLocales = () => {
 };
 
 const getTemplate = (page) => {
-  const filepath = path.join(__dirname, '..', 'templates', 'login', templates[page].file);
-  templates[page].content = template.getTemplate(filepath);
-  return templates[page].content;
+  return template.getTemplate(templates[page].file);
```

Comment:
This was caching the result of the getTemplate call but the `template` service already caches it so this helped clean things up a little.